pkg/azurefile/azurefile.go

```diff
@@ -870,6 +881,60 @@ func (d *Driver) ResizeFileShare(ctx context.Context, subsID, resourceGroup, acc
 	})
 }
 
+// CopyFileShare copies a fileshare in the same storage account
+func (d *Driver) CopyFileShare(ctx context.Context, req *csi.CreateVolumeRequest, accountKey string, shareOptions *fileclient.ShareOptions, storageEndpointSuffix string) error {
```

Comment:
for the non CSI functions, let's make it internal func:
`CopyFileShare` --> `copyFileShare`

Reply:
done